backend/docker-compose.yml

```
@@ -18,6 +18,17 @@ services:
       timeout: 5s
       retries: 10
 
+  #---------------------------------------------
+  # Historic data
+  #---------------------------------------------
+  historic-data:
```

```suggestion
  historic-data:
```
This needs to be added to both docker compose files